serverless/cluster-logging-serverless.adoc

```diff
@@ -6,6 +6,12 @@ include::modules/serverless-document-attributes.adoc[]
 
 toc::[]
 
+[IMPORTANT]
+====
+You are viewing documentation for a release of Red Hat OpenShift Serverless is no longer supported.
```

Comment:
Red Hat OpenShift Serverless `that` is no longer supported? I'm curious if something is missing in the sentence.

And same comment for all the occurrences of this text.

Reply:
LOL good thing you read it, I'll add "that" for sure. Thanks!